keyboards/albert_smart/info.json

```diff
@@ -0,0 +1,12 @@
+{
+    "manufacturer": "albert-zheng",
+    "maintainer": "albert-zheng",
+    "url": "",
+    "bootloader": "stm32duino",
+    "bootloader_instructions": "Press ESC(ROW0COL0), Plug USB. ",
+    "processor": "STM32F103",
+    "board": "STM32_F103_STM32DUINO",
```

Comment:
This is information that is generally much more keyboard specifically, and shouldn't really be at the a vendor level.

Reply:
Uh, should I delete it or add it to readme file?